src/os/interaction/modifyinteraction.js

```diff
@@ -36,19 +137,51 @@ const OSMap = goog.requireType('os.Map');
 class Modify extends OLModify {
   /**
    * Constructor.
-   * @param {olx.interaction.ModifyOptions=} opt_options Options.
+   * @param {!Feature} feature The feature to modify.
    */
-  constructor(opt_options) {
-    const options = opt_options || {};
+  constructor(feature) {
+    const interpolationMethod = /** @type {interpolate.Method} */ (feature.get(interpolate.METHOD_FIELD)) ||
+        interpolate.Method.RHUMB;
```

Comment:
Should this default to the current application interpolation method instead of Rhumb?

Reply:
Yeah I'm not sure why I defaulted to rhumb. It's now using `interpolate.getMethod()` instead.